I.write-tests/09-test-async/async-1.test.js

```diff
@@ -1,11 +1,11 @@
 const getRepos = require("./async-1");
 
 test("gets a list of repositories names", function() {
-  // arrange
+  //arrange
```

Comment:
I don't think this change is necessary 😊 We can remove all these cosmetic changes what do you think?